frontend/packages/dev-console/src/components/catalog/PinnedResourcesConfiguration.tsx

```diff
@@ -0,0 +1,340 @@
+import * as React from 'react';
+import { DualListSelector, FormHelperText, FormSection, Tooltip } from '@patternfly/react-core';
+import * as fuzzy from 'fuzzysearch';
+import { Map as ImmutableMap, Set as ImmutableSet } from 'immutable';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import {
+  DiscoveryResources,
+  ExtensionK8sModel,
+  K8sKind,
+  K8sModel,
+  ResourceIcon,
+} from '@console/dynamic-plugin-sdk/src/lib-core';
+import {
+  K8sResourceKind,
+  modelFor,
+  referenceForGroupVersionKind,
+  referenceForModel,
+} from '@console/internal/module/k8s';
+import { RootState } from '@console/internal/redux';
+import { YellowExclamationTriangleIcon, useTelemetry } from '@console/shared';
+import {
+  useDebounceCallback,
+  useConsoleOperatorConfig,
+  LoadError,
+  SaveStatus,
+  SaveStatusProps,
+  patchConsoleOperatorConfig,
+} from '@console/shared/src/components/cluster-configuration';
+import {
+  Perspective,
+  PerspectivePinnedResource,
+  PerspectiveVisibilityState,
+  usePerspectives,
+} from '@console/shared/src/hooks/perspective-utils';
+import './PinnedResourcesConfiguration.scss';
+
+// Blacklist known duplicate resources.
+const blacklistGroups = ImmutableSet([
```

Comment:
We should avoid a nomenclature like this, please use a term like `skipDuplicateGroups` or `hideGroups`.

And is this really needed at all if you only show primary CRDs?

Reply:
I changed the wordings. I followed the same logic used in Search page.